### ISSUE TITLE
Overfitting on the training dataset

### ISSUE THREAD

**Issue by greatlog** [goutamgmb/deep-burst-sr#22]
Thanks for your wonderful work. I have some problems when I try to fine-tune DBSR on the BurstSR dataset for a burst size of 14. 

I firstly train it on the synthetic dataset and it finally achieves 40.95 dB on the synthetic validation dataset. Then I try to fine-tune it on the BurstSR dataset. While the validation result only grows at the beginning but drops gradually after it achieves 47.86 dB. It seems that the training set of BurstSR is overfitted.

Have you met the same problem? Is it because the training set is too small or there is a domain gap between the training and validation set?  I have been confused by this problem for months. I would appreciate it a lot if you could help me figure this out.

Thanks.

**Comment by goutamgmb** [comment]
Hi,

Firstly the training set of BurstSR is obviously smaller compared to the synthetic training dataset. Secondly, while the AlignedL2 loss tries to handle the color and spatial mis-alignments, its is not fully accurate (compared to pixel-accurate ground truth available in synthetic training). This can also lead to saturation of validation performance in my opinion.

**Comment by greatlog** [comment]
Hi, thanks for your reply. 

I agree with your opinion. So is there any method to alleviate the overfitting? Because of the overfitting, I can not re-implement your results of DBSR with burst size as 14. Would you mind sharing with me some tricks about it?

Thanks. Looking forward to your reply.

**Comment by goutamgmb** [comment]
Hi,

You should be able to reproduce the original results, or get similar results, by training with the [default parameters](https://github.com/goutamgmb/deep-burst-sr/blob/master/train_settings/dbsr/default_realworld.py) as it is. What results are you getting currently?

Regards

**Comment by goutamgmb** [comment]
Hi,

I doubt there is a bug if the results are matching for burst size 8. The behavior could just be due to some randomness in training. 

Regards

**Comment by greatlog** [comment]
I get it. 

Thanks for your sharing!
